relecov_tools/assets/schema_utils/custom_validators.py

```
@@ -31,4 +31,4 @@ def validate_with_exceptions(schema, data, errors):
         # Keep all other errors
         filtered_errors.append(error)
 
-    return
+    return filtered_errors
```

You should update the description as it now returns something

---

Thanks! i'll change the description. This asset is used on `json_validation.py` and should return the validation errors found.